packages/server/src/Document.ts

```diff
@@ -260,6 +260,7 @@ export class Document extends Doc {
 
   destroy() {
     super.destroy()
+    this.awareness.destroy()
```

Comment:
this might fix pending _checkInterval timers

Reply:
ok probably not, as y-protocols/awareness destroys itself when the attached doc is destroyed.